Tests/Unit/MSFT_xIISFeatureDelegation.tests.ps1

```diff
@@ -4,14 +4,49 @@ $global:DSCResourceName = 'MSFT_xIISFeatureDelegation'
 #region HEADER
 
 [String] $moduleRoot = Split-Path -Parent (Split-Path -Parent (Split-Path -Parent $Script:MyInvocation.MyCommand.Path))
- if ( (-not (Test-Path -Path (Join-Path -Path $moduleRoot -ChildPath 'DSCResource.Tests'))) -or `
-      (-not (Test-Path -Path (Join-Path -Path $moduleRoot -ChildPath 'DSCResource.Tests\TestHelper.psm1'))) )
+$repoSource = (Get-Module -Name $global:DSCModuleName -ListAvailable)
+
+if (($repoSource -ne $null) -and ($repoSource[0].RepositorySourceLocation.Host -eq 'www.powershellgallery.com'))
+{
+    if ( -not (Test-Path -Path (Join-Path -Path $moduleRoot -ChildPath 'Tests\DscResourceTestHelper')) )
+    {
+
+        $choice = read-host "In order to run this test you need to install a helper module, continue with installation? (Y/N)"
```

Comment:
This will only come up if they're installing the module from the gallery - and only the first time the user runs the tests. We just didn't want to automatically being downloading files onto their computer